lib/active_merchant/billing/gateways/litle.rb

```diff
@@ -561,7 +561,7 @@ def commit(kind, request, money = nil)
       end
 
       def success_from(kind, parsed)
-        return %w(000 001 010).any?(parsed[:response]) unless kind == :registerToken
+        return %w(000 001 010 141 142).any?(parsed[:response]) unless kind == :registerToken
```

Comment:
Do you have some documentation to support this?

Reply:
Nope, the documentation we have doesn't mention these two response codes. We do have example transactions and they literally do say Approved at the end of the response message. If they are failed it says Soft Decline at the end and the response code for that is 138.

Reply:
I remember some gateway as mentioning one message as "Approved" but the transaction code indicated it failed, do we think this gateway might be the same?

Reply:
Not in this case, the customer did reach out to Litle and they confirmed that those codes would be considered successful.